datasetUtils.py

```diff
@@ -391,6 +391,68 @@ def mergeDatasets(dataset1, dataset2, output):
             else:
                 f.write(line)
 
+    # append a 'taco status' to the data.yaml file, to know which dataset was used
+    with open(output + "/data.yaml", "a") as f:
+        f.write("\n#dataset status\n")
+        f.write("taco: 3 split\n")
+        f.write("roboflow: see above\n")
+
+def mergeTacoDatasetAsTrain(sourceDataset, tacoPath, output):
+    """Merges two datasets to the output directory."""
+
+    if not os.path.exists(output):
+        os.makedirs(output)
+
+    # Copy the datasets val, train and test directories to the output directory
+    for directory in ["train", "test", "val"]:
+        if not os.path.exists(output + "/" + directory):
+            os.makedirs(output + "/" + directory)
+        if os.path.exists(sourceDataset + "/" + directory):
+
+            for type in ["images", "labels"]:
+                if not os.path.exists(output + "/" + directory + "/" + type):
+                    os.makedirs(output + "/" + directory + "/" + type)
+                accdir = '/' + directory + '/' + type
+                for file in os.listdir(sourceDataset + accdir):
+                    shutil.copy(sourceDataset + accdir + "/" + file, output + accdir + "/" + file)
+        
+        # copy the second dataset, if it exists and moving everything from dataset2/val to output/val, knowing output/val already exists
+        if os.path.exists(tacoPath + "/" + directory):
+            for type in ["images", "labels"]:
+                if not os.path.exists(output + "/" + directory + "/" + type):
+                    os.makedirs(output + "/" + directory + "/" + type)
+                accdir = '/' + directory + '/' + type
+
+                for file in os.listdir(tacoPath + accdir):
+                    shutil.copy(tacoPath + accdir + "/" + file, output + "/train/" + type + "/" + file)
```

Comment:
Here is the 'Taco as train' part - when merging the datasets, and getting all images and labels from TACO/train, TACO/test, TACO/val, I put them all in output/train rather then in their base TACO directory